### PR TITLE
Sourcegraph badge

### LINKED ISSUE #104: Sourcegraph badge
As seen in [pgrok](https://github.com/pgrok/pgrok), sourcegraph badge can be added into README.

```
[![Sourcegraph](https://img.shields.io/badge/view%20on-Sourcegraph-brightgreen.svg?style=for-the-badge&logo=sourcegraph)](https://sourcegraph.com/github.com/sadikkuzu/squiral)
```

[Sourcegraph](https://sourcegraph.com/github.com/sadikkuzu/squiral)

### PR DESCRIPTION
**Pull request by ayushmaanshrotriya**
added Sourcegraph badge to readme.md

### CODE REVIEW

**Comment by ayushmaanshrotriya** [comment]
#104

File: README.md [hunk 1]
```diff
@@ -8,6 +8,8 @@
 [![Python lint&test](https://github.com/sadikkuzu/squiral/actions/workflows/python-package.yml/badge.svg)](https://github.com/sadikkuzu/squiral/actions/workflows/python-package.yml)
 [![Publish Python Package](https://github.com/sadikkuzu/squiral/actions/workflows/python-publish.yml/badge.svg)](https://github.com/sadikkuzu/squiral/actions/workflows/python-publish.yml)
```
Comment (sadikkuzu) [inline README.md:10]: Could you add `<br/>` into this 10th line?

Comment (ayushmaanshrotriya) [reply README.md]: done

Comment (sadikkuzu) [reply README.md]: Cool, thanks 👍🏼

**Pull request merged**